Makefile

```diff
@@ -1,17 +1,37 @@
 REPO := quay.io/boson/faas
 BIN  := faas
 
+WINDOWS=$(BIN)_windows_amd64.exe
+LINUX=$(BIN)_linux_amd64
+DARWIN=$(BIN)_darwin_amd64
```

Comment:
While this may sound pedantic, I would consider putting these in alphabetical order to avoid any misunderstanding that we have a preference of one platform over another.